modules/learn/pages/clusters-and-availability/rebalance-and-index-service.adoc

```diff
@@ -0,0 +1,838 @@
+= Index Rebalance
+:description: pass:q[This page explains how rebalance operations impact the Index Service in Couchbase Server, covering file-based rebalance, shard affinity, index redistribution, and node failover.]
+:page-aliases: clusters-and-availability:rebalance
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+Rebalance affects different services differently.
+The following sections explain how rebalance operation affects the Index Service.
+
+== Index Service
+
+The Index Service maintains cluster-wide index definitions and metadata to <<#index-redistribution,redistribute indexes and replicas>> during rebalance operations.
+
+During rebalance, Couchbase Server evaluates each node’s CPU, RAM, disk bandwidth, and user-defined Server Groups (Availability Zones) to minimize database performance impact and maximize index availability.
```

Comment:
Instead of saying "minimize database performance impact", we can just say "improve performance"